packages/yourloops/pages/profile/profile.tsx

```diff
@@ -307,7 +308,9 @@ export const ProfilePage: FunctionComponent = () => {
       <Snackbar params={snackbarParams} />
       <Container className={classes.container} maxWidth="sm">
         <div style={{ display: "flex", flexDirection: "column", margin: "16px" }}>
-          <DialogTitle className={classes.title}>{t("account-preferences-title")}</DialogTitle>
+          <div className={classes.title} id="account-preferences-title">
```

Comment:
No need to change the `<DialogTitle/>` with a `<div/>` here, you can put the id in it.